content/applications/inventory_and_mrp/inventory/barcode/operations/transfers_scratch.rst

```diff
@@ -1,17 +1,200 @@
+options is presented. To process internal transfers, click on the :guilabel:`Operations` button at
+the bottom of the screen. This navigates to an :menuselection:`Operations` overview page.
+
+.. image:: transfers_scratch/transfers-scratch-barcode-app.png
+   :align: center
+   :alt: Barcode app start screen with scanner.
+
+From this page, locate the :guilabel:`Internal Transfers` card, and click the :guilabel:`# To
+Process` button to view all outstanding internal transfers. Then, select the desired operation to
+process. This navigates to the barcode transfer screen.
+
+.. note::
+   If *only* using a barcode scanner or the Odoo mobile app, the barcodes for each transfer of a
+   corresponding operation type can be scanned to be processed easily. Once scanned, the products
+   that are part of an existing transfer can be scanned, and new products can be added to the
+   transfer, as well. Once all products have been scanned, validate the transfer to proceed with the
+   stock moves.
+
+From this screen, an overview of all products to process within that specific internal transfer
+(**WH/INT/000XX**) is shown. At the bottom of the screen, there are options to :guilabel:`Add
+Product` or :guilabel:`Validate`, depending on if products need to be added to the operation, or if
+the whole operation should be validated at once.
+
+.. image:: transfers_scratch/transfers-scratch-receipts-overview.png
+   :align: center
+   :alt: Overview of receipts in transfer to scan.
+
+To process and scan each product individually, choose a specific product line. The :guilabel:`+#`
+button (in this case, :guilabel:`+1`) can be clicked to add additional quantity of that product to
+the transfer, or the :guilabel:`pencil icon` can be clicked to open a new screen to edit that
+product line.
+
+From this screen, the product that's being transferred is listed. Under the product name, the
+:guilabel:`Quantity` line can be edited. Change the number in the line to the quantity listed to be
+transferred on the internal transfer form.
+
+.. example::
+   In the internal transfer operation `WH/INT/000XX`, `50 Units` of the `Transfer Product` is
+   expected to be transferred. `[TRANSFER_PROD]` is the :guilabel:`Internal Reference` set on the
+   product form. Scan the barcode of the `Transfer Product` to receive one unit. Afterwards, click
+   the :guilabel:`pencil icon` to manually enter the transferred quantities.
+
+   .. image:: transfers_scratch/transfers-scratch-product-line-editor.png
+      :align: center
+      :alt: Product line editor for individual transfer in Barcode app.
+
+Additionally, the :guilabel:`+1` and :guilabel:`-1` buttons can be clicked to add or subtract
+quantity of the product, and the :guilabel:`number keys` can be used to add quantity, as well.
+
+Below the :guilabel:`number keys` are the two :guilabel:`location` lines, which read `WH/Stock` by
+default, or whichever locations were previously specified on the internal transfer form. Click these
```

Comment:
> which read `WH/Stock` by default

Is incorrect, to my knowledge :astonished: 
The location is whatever the source location is on the Internal transfer form
![image](https://github.com/odoo/documentation/assets/23611711/07d7c350-003d-4086-b2d8-5a476bb40cc0)
![image](https://github.com/odoo/documentation/assets/23611711/e1c4d3e7-31ab-4776-895c-e152e10552e3)

Reply:
I *think* you were trying to address the `WH/Stock` in the image, and I think the best way to directly refer to the WH/Stock source location is to explain that WH/Stock is the source location in an example admonition

Reply:
Good catch! Changed the wording here to explain that